common/src/main/java/io/druid/math/expr/Expr.java

```diff
@@ -19,15 +19,17 @@
 
 package io.druid.math.expr;
 
+import com.google.common.base.Strings;
 import com.google.common.math.LongMath;
 
 import java.util.List;
+import java.util.Objects;
 
 /**
  */
 public interface Expr
 {
-  Number eval(ObjectBinding bindings);
+  ExprEval eval(ObjectBinding bindings);
```

Comment:
Can we keep this? It's trivial but makes so many conflicts in following PRs. We can discuss anytime later.